src/store/index.ts

```diff
@@ -37,23 +37,30 @@ const {
     currentWorkspacePermission: null,
   } as State,
   getters: {
+    tables(state: State, getters): string[] {
+      if (state.currentWorkspace !== null && state.currentWorkspace.nodeTables && state.currentWorkspace.edgeTables) {
+        return getters.nodeTables.concat(getters.edgeTables).sort();
+      }
+      return [];
```

Comment:
What problem does it solve to alphabetize all the tables? There's an argument that the tables should appear in order of newness (though I'm not sure that's what happens currently), but alphabetic is a fine default choice too.

More generally, we ought to be able to change the sort order on the fly. Would you mind filing an issue to that effect?

Reply:
I was still curious why you wanted to see tables alphabetized.